test_scripts/API/Restructuring_OnResetTimeout/016_BC_OnResetTimeout_with_wrong_requestID_and_HMI_did_not_respond.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 3000) with right methodName but wrong requestID to SDL
+-- 4) HMI does not send response
+-- SDL does:
+-- 1) Respond in 10 seconds with GENERIC_ERROR resultCode to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function scrollableMessageError( )
+  local requestParams = {
+    scrollableMessageBody = "abc",
+    timeout = 5000
+  }
+  local cid = common.getMobileSession():SendRPC("ScrollableMessage", requestParams)
+  EXPECT_HMICALL("UI.ScrollableMessage",
+    { messageText = {
+      fieldName = "scrollableMessageBody",
+      fieldText = requestParams.scrollableMessageBody
+    },
+    appID = common.getHMIAppId()
+  })
+  :Do(function()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnResetTimeout",
+    { requestID = 111,
+    methodName = "ScrollableMessage",
```

Comment:
methodName and requestID are invalid, invalid values must be checked separately, not all in one step.